_pkgdown.yml

```diff
@@ -1,15 +1,19 @@
 reference:
+- title: Create or learn about local repositories.
+  contents:
+  - '`repository`'
+  - '`git_open`'
```

Comment:
I normally would have just listed `- git_open` here, I think. What's going on with the single quotes + back ticks here?